apps/app/src/app/pages/(components)/components/(data-table)/data-table.preview.ts

```diff
@@ -302,8 +309,10 @@ export class DataTablePreviewComponent {
 	private readonly _debouncedFilter = toSignal(toObservable(this._rawFilterInput).pipe(debounceTime(300)));
 
 	private readonly _displayedIndices = signal({ start: 0, end: 0 });
-	protected readonly _availablePageSizes = [5, 10, 20, 10000];
-	protected readonly _pageSize = signal(this._availablePageSizes[0]);
+	protected readonly _availablePageSizes = [5, 10, 20, 200, 10000];
+
+	protected readonly _defaultValue = this._availablePageSizes[0];
+	protected readonly _pageSize = signal(this._availablePageSizes[0].toString());
```

Comment:
Let's keep number here

Reply:
Thanks @goetzrobin for your review, I realized by letting as number the default value was not beeing defined

That was my previous code:

```javascript	
protected readonly _availablePageSizes = [5, 10, 20, 200, 10000];
protected _pageSize = signal(this._availablePageSizes[0]);

// The template
<hlm-select class="mr-1 inline-block" [(ngModel)]="_pageSize" name="page-size">
  <hlm-select-trigger class="h-9 w-20 py-0">
	  <brn-select-value hlm />
  </hlm-select-trigger>
  <hlm-select-content class="w-20 min-w-0 space-y-1">
	  @for (size of _availablePageSizes; track size) {
		  <hlm-option value={{size}}>{{ size === 10000 ? 'All' : size }}</hlm-option>
	  }
  </hlm-select-content>
</hlm-select>
```

and the corresponding result on the select:

![image](https://github.com/goetzrobin/spartan/assets/31545128/b8d1fc2d-bdc9-4296-b673-c97100af28d3)


I'm considering the following change, by creating a sepecific computed signal to paginator:

```javascript	

// template
<div
	class="mt-4 flex flex-col justify-between sm:flex-row sm:items-center"
	*brnPaginator="let ctx; totalElements: _totalElements(); pageSize: _pageSizePaginator(); onStateChange: _onStateChange"
>

// ts
protected _pageSize = signal(this._availablePageSizes[0].toString());
protected _pageSizePaginator = computed(() => Number(this._pageSize()));
```

Reply:
It is actually a bug that should be solved on the select component.